tests/testthat/test-trace-collapse-eval.txt

```diff
@@ -4,17 +4,13 @@ Full:
   ├─base::eval(quote(g()))
   │ └─base::eval(quote(g()))
   └─rlang:::g()
```

Comment:
Can you explain why these are now trimmed?

Reply:
Because these are sibling nodes:

```
█
└─global::f()
  ├─base::eval(quote(g()))
  │ └─base::eval(quote(g()))
  └─global::g()
    ├─base::eval(quote(trace_back(e, 0)))
    │ └─base::eval(quote(trace_back(e, 0)))
    └─rlang::trace_back(e, 0)
```

I have set `bottom` to 0 to preserve the original test output.